logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/JoinExistingMLSConversationsUseCase.kt

```diff
@@ -65,7 +65,11 @@ internal class JoinExistingMLSConversationsUseCaseImpl(
                                 )
                                 Either.Right(Unit)
                             } else {
-                                Either.Left(it)
```

Comment:
can we filter by specific error that we need to ignore?

Reply:
According to user story, we should not propagate non-recoverable errors from this use case.
https://wearezeta.atlassian.net/browse/WPB-5155

Reply:
@ohassine can you please add a comment that this is a non recoverable error

Reply:
@ohassine but network errors are recoverable so can we at least filter out if the user is offline